app/views/ServiceConfigPage.scala.html

```diff
@@ -29,9 +29,11 @@
   request: RequestHeader
 )
 
-@display(env: String, textIfTrue: String, textIfFalse: String = "") = {
-    @defining(deployments.map(_.environment.asString) :+ "local"){deployedEnvs =>
-        @if(!deployedEnvs.contains(env)){ @textIfTrue } else { @textIfFalse }
+@hasDeployment(env: ConfigEnvironment, textIfTrue: String, textIfFalse: String = "") = @{
+    if ((deployments.map(_.environment.asString) :+ "local").contains(env.asString)) {
```

Comment:
Could avoid comparing the `asString` representation with `(deployments.map(ConfigEnvironment.ForEnvironment.apply) :+ ConfigEnvironment.Local).contains(env)`

Reply:
Providing both `textIfTrue` and `textIfFalse` runs the risk of being mixed up, at least harder to follow (note, it's being called without named params).
I think using `@defining(..){deployments =>` which would move the boolean logic to the call points could be simpler? e.g. `@if(deployments.contains(env) { "hidden"}`

Reply:
Added named parameters as discussed

Reply:
Refactored to compare types not Strings